### PR TITLE
[Feature: GradeInquiry] Added count of grade inquiries to lateday table

### LINKED ISSUE #2336: Add statistics on grade inquiry to student info lateday table
Add a new column (and maybe a counter) to show whether or not the student has requested a regrade.

This will help identify patterns of chronic grade inquiry / unreasonable regrade requests.

### PR DESCRIPTION
**Pull request by sak6lab**
### What is the current behavior?
Late day table does not include information of grade inquiry history for the student
closes #2336 

### What is the new behavior?
Late day table now shows how many grade inquiries there are per gradeable pending or resolved
grade-inquiry-late-day-table

### CODE REVIEW

File: site/app/templates/LateDaysTablePlugin.twig [hunk 1]
```diff
@@ -27,6 +27,7 @@
             <th>Assignment submitted # of days after deadline</th>
             <th>Student granted # of days extension for this assignment</th>
             <th>Status</th>
+            <th>Grade Inquiries</th>
```
Comment (zjxiaohan) [inline site/app/templates/LateDaysTablePlugin.twig]: It is better to only show this column when grade inquiry is enabled in the course.

**Comment by sak6lab** [comment]
@zjxiaohan This information can be shown in the student information panel for the electronic grader interface. Graders can then see grade inquiries made for other gradeables. Perhaps, this column should only be shown for graders?

File: site/app/templates/LateDaysTablePlugin.twig (hunk 1 above)
Comment (sak6lab) [reply site/app/templates/LateDaysTablePlugin.twig]: Will do!

**Comment by zjxiaohan** [comment]
> @zjxiaohan This information can be shown in the student information panel for the electronic grader interface. Graders can then see grade inquiries made for other gradeables. Perhaps, this column should only be shown for graders?

Oh yeah I forgot that graders can see the late day table! :)

**Pull request merged**